src/components/Box/index.js

```diff
@@ -1,19 +1,34 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
 import { x } from '@xstyled/emotion'
 import PropTypes from 'prop-types'
 import React, { forwardRef } from 'react'
 
-const Box = forwardRef(({ width, height, ...props }, ref) => (
-  <x.div ref={ref} w={width} h={height} {...props} />
+const borderedStyles = ({ theme }) => css`
+  padding: ${theme.space['3']};
+  border-radius: ${theme.radii.default};
+  border: 1px solid ${theme.colors.gray350};
+`
+
+const StyledBox = styled(x.div, {
+  shouldForwardProp: prop => !['bordered'].includes(prop),
+})`
+  ${({ bordered }) => (bordered ? borderedStyles : null)}
+`
+const Box = forwardRef(({ width, height, bordered, ...props }, ref) => (
+  <StyledBox ref={ref} w={width} h={height} bordered={bordered} {...props} />
```

Comment:
If you pass the class directly would this avoid the noise in the snapshots ?

```js
    <w.div ref={ref} w={width} h={height} css={bordered && borderedStyles} {...props} />
```

Reply:
probleme with having the css class directly instead of styled component is the overwriting. With a css class directly then the `p, border, etc.` have no effect on what's inside the borderedStyles. So you can't overwrite the padding and border. Which is a problem I reckon :(

Reply:
Alrighty then :)